shared/chat/conversation/messages/message-popup/payment/index.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import * as React from 'react'
+import {toUpper, upperFirst} from 'lodash-es'
+import * as Kb from '../../../../../common-adapters'
+import * as S from '../../../../../styles'
```

Comment:
Are we doing `S` now instead of `Styles`? Wow I'm so behind on what's hip already

Reply:
we should use Styles i think

Reply:
Ok :(